lib/src/main/java/org/esbtools/eventhandler/PollingDocumentEventProcessorRoute.java

```diff
@@ -51,12 +52,24 @@ public PollingDocumentEventProcessorRoute(DocumentEventRepository documentEventR
         this.batchSize = batchSize;
         this.documentEndpoint = documentEndpoint;
         this.failureEndpoint = failureEndpoint;
+        this.routeId = "documentEventProcessor-" + id;
```

Comment:
Let's make this `this.routeId = "documentEventProcessor-" + idCounter.getAndIncrement();` and remove the `int id` field since the name doesn't quite make as much sense now.